packages/backend/package.json

```diff
@@ -9,7 +9,7 @@
     "prebuild": "rimraf dist",
     "build": "nest build",
     "format": "prettier --write \"src/**/*.ts\" \"test/**/*.ts\"",
-    "start": "cross-env NODE_ENV=production nest start",
+    "start": "nest start",
```

Comment:
is defaulted to prod in code?

Reply:
yup

Reply:
if it doesn't see development explicitly it does prod

Reply:
also this way we don't have to list cross-env as a non-dev dependency